frontend/lib/swr-cache/adapters/index.ts

```diff
@@ -11,7 +11,10 @@ export const getCache = () => {
    const client = new Redis(REDIS_URL, {
       connectTimeout: 500,
       // Retry, connect every once in a while as cache misses / failures are OK
-      retryStrategy: (times) => Math.min(times * 5000, 10 * 60 * 1000),
+      retryStrategy: (times) => {
+         times = Math.min(20, times);
+         return Math.min((50 + 2) ** (times + 3), 10 * 60 * 1000);
```

Comment:
What is this backoff supposed to be? 52**3 = 140608 which is over 2 mins if times == 0. If times == 1 it's 7311616 which is already over the 10 min cap. I think we should probably max out at 1 minute, maybe even 30 sec. I'm not really worried about spamming redis with connections if they're not going through. We should probably have this retry in under 1 s (maybe 100ms) for the first failure and then if we care, we can back it off from there.

Reply:
Well crap, I could have sworn that's not what I typed, or at least not what I typed at first...

Well what do you know... prettier screwed it up: https://github.com/iFixit/react-commerce/pull/1341/commits/9f50df6653c618e507478fe402222675159aba5e

Back when it was XOR, it helpful re-inforced the natural evaluation order for bitwise operators I guess... or something. Fixing. Sheesh.

Reply:
```suggestion
         return Math.min(50 + (2 ** (times + 3)), 2 * 60 * 1000);
```